packages/babel-plugin-transform-destructuring/test/fixtures/destructuring/array-unpack-optimisation/input.js

```diff
@@ -11,3 +11,7 @@ var [a, b] = [...foo, bar];
 var [a, b] = [foo(), bar];
 var [a, b] = [clazz.foo(), bar];
 var [a, b] = [clazz.foo, bar];
+var [a, b] = [, 2];
+[a, b] = [1, 2];
+[a, b] = [, 2];
+; // Avoid completion record special case
```

Comment:
Not in this PR, but this optimization is wrong: `;` keeps the previous completion value:
```js
eval("0;;;;")
```

Can you use something like `0;` instead?

Reply:
Yeah, I've just realised this on another PR. Ultimately it's a false negative in the `isCompletionRecord` check, which could be quite expensive to correct without memoising somehow.

I'll change this, but FWIW there are several other tests that currently use `;` in this way and would need to change similarly.

Reply:
This can/should be handled in another PR